test/powershell/Provider/ProviderIntrinsics.Tests.ps1

```diff
@@ -5,19 +5,14 @@ Describe "ProviderIntrinsics Tests" -tags "CI" {
         setup -d TestDir
     }
     It 'If a childitem exists, HasChild method returns $true' {
-        $ExecutionContext.InvokeProvider.ChildItem.HasChild("$TESTDRIVE") | Should be $true
+        $ExecutionContext.InvokeProvider.ChildItem.HasChild("$TESTDRIVE") | Should -BeTrue
     }
     It 'If a childitem does not exist, HasChild method returns $false' {
-        $ExecutionContext.InvokeProvider.ChildItem.HasChild("$TESTDRIVE/TestDir") | Should be $false
+        $ExecutionContext.InvokeProvider.ChildItem.HasChild("$TESTDRIVE/TestDir") | Should -BeFalse
     }
     It 'If the path does not exist, HasChild throws an exception' {
-        try {
-            $ExecutionContext.InvokeProvider.ChildItem.HasChild("TESTDRIVE/ThisDirectoryDoesNotExist")
-            throw "Execution OK"
-        }
-        catch {
-            $_.fullyqualifiederrorid | should be "ItemNotFoundException"
-        }
+        {$ExecutionContext.InvokeProvider.ChildItem.HasChild("TESTDRIVE/ThisDirectoryDoesNotExist")} |
```

Comment:
nit: please add whitespace after open brace and before closing brace.  same for other ones below.

Reply:
I made this correction to this file and 1 other that had the same issue.